### PR TITLE
Update repo URL in `package.json`

### PR DESCRIPTION
**Pull request by Gudahtt**
It had not been updated when it was transferred to the MetaMask organization.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com/flyswatter/eth-token-tracker.git"
+    "url": "git+ssh://git@github.com/metamask/eth-token-tracker.git"
```
Comment (whymarrh) [inline package.json]:
```suggestion
    "url": "git+ssh://git@github.com/MetaMask/eth-token-tracker.git"
```

File: package.json [hunk 2]
```diff
@@ -21,9 +21,9 @@
   "author": "Dan Finlay <dan@danfinlay.com>",
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/flyswatter/eth-token-tracker/issues"
+    "url": "https://github.com/metamask/eth-token-tracker/issues"
```
Comment (whymarrh) [inline package.json]:
```suggestion
    "url": "https://github.com/MetaMask/eth-token-tracker/issues"
```

File: package.json [hunk 3]
```diff
@@ -21,9 +21,9 @@
...
   },
-  "homepage": "https://github.com/flyswatter/eth-token-tracker#readme",
+  "homepage": "https://github.com/metamask/eth-token-tracker#readme",
```
Comment (whymarrh) [inline package.json]:
```suggestion
  "homepage": "https://github.com/MetaMask/eth-token-tracker#readme",
```

File: package.json (hunk 1 above)
Comment (Gudahtt) [reply package.json]: Whoops! Fixed.

**Pull request merged**